sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationSettingPage.java

```diff
@@ -30,12 +31,12 @@ public String getContinuationToken() {
     }
 
     /**
-     * Gets the list of {@link ConfigurationSetting ConfigurationSettings} on this page.
+     * Gets the iterable stream of {@link ConfigurationSetting ConfigurationSettings} on this page.
      *
-     * @return The list of {@link ConfigurationSetting ConfigurationSettings}.
+     * @return The iterable stream of {@link ConfigurationSetting ConfigurationSettings}.
      */
     @Override
-    public List<ConfigurationSetting> getItems() {
-        return items;
+    public IterableStream<ConfigurationSetting> getElements() {
+        return items == null ? IterableStream.empty() : new IterableStream<>(items);
```

Comment:
added `IterableStream.of(Iterable<T>)` factory method:  if parameter to this factory is `null` then it return cached static empty stream otherwise it create a new `IterableStream` from the parameter.